README.md

```diff
@@ -47,6 +47,10 @@ go get -u github.com/snowflakedb/gosnowflake
 For detailed documentation and basic usage examples, please see the documentation at
 [godoc.org](https://godoc.org/github.com/snowflakedb/gosnowflake/).
 
+## Note
```

Comment:
What is the context of this note? I'm wondering if we should add this note here or in doc.go.

Reply:
The same note in README we are adding to all our opensource drivers